src/components/Icon/icons/index.tsx

```diff
@@ -45,30 +45,6 @@ export const FailedIcon = () => (
   </svg>
 );
 
-export const RunningIcon = () => (
```

Comment:
This is depended on by a [component](https://github.com/evergreen-ci/spruce/blob/main/src/components/PatchesPage/patchCard/BuildStatusIcon.tsx#L37) We still need this Icon or need to replace it where we use it.

Reply:
This is why the tests  are failing.